react/features/conference/components/native/AlwaysOnLabels.tsx

```diff
@@ -28,35 +30,41 @@ interface IProps {
 
 const AlwaysOnLabels = ({ createOnPress }: IProps) => {
     const dispatch = useDispatch();
+    const isStreaming = useSelector((state: IReduxState) => Boolean(getActiveSession(state, JitsiRecordingConstants.mode.STREAM)));
     const openHighlightDialogCallback = useCallback(() => dispatch(openHighlightDialog()), [ dispatch ]);
 
-    return (<>
-        <TouchableOpacity
-            hitSlop = { LabelHitSlop }
-            onPress = { createOnPress(LABEL_ID_RECORDING) } >
-            <RecordingLabel mode = { JitsiRecordingConstants.mode.FILE } />
-        </TouchableOpacity>
-        <TouchableOpacity
-            hitSlop = { LabelHitSlop }
-            onPress = { createOnPress(LABEL_ID_STREAMING) } >
-            <RecordingLabel mode = { JitsiRecordingConstants.mode.STREAM } />
-        </TouchableOpacity>
-        <TouchableOpacity
-            hitSlop = { LabelHitSlop }
-            onPress = { openHighlightDialogCallback }>
-            <HighlightButton />
-        </TouchableOpacity>
-        <TouchableOpacity
-            hitSlop = { LabelHitSlop }
-            onPress = { createOnPress(LABEL_ID_RAISED_HANDS_COUNT) } >
-            <RaisedHandsCountLabel />
-        </TouchableOpacity>
-        <TouchableOpacity
-            hitSlop = { LabelHitSlop }
-            onPress = { createOnPress(LABEL_ID_VISITORS_COUNT) } >
-            <VisitorsCountLabel />
-        </TouchableOpacity>
-    </>);
+    return (
+        <>
```

Comment:
Please avoid this cosmetic change. It makes the whole function looks as though it changed, making it harder to review.